assets/scss/_changelog.scss

```diff
@@ -0,0 +1,9 @@
+.changelog-fix {
+    background-color: #F5436A;
+}
+.changelog-enhancement {

```

Comment:
These are not used currently but maybe we want them to be used / customized. Also the integrated theme ones might change. Also we might want to change them when switching the theme (dark/light).

Reply:
Yes, maybe we want to be able to customize them, but as we currently don't have any particular colors  as design I'll simply remove them.